examples/spark/spark-worker-controller.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: v1
 metadata:
   name: spark-worker-controller
 spec:
-  replicas: 3
+  replicas: 1
```

Comment:
the goal is to minimize flakes and test a distilled cluster.  
2 is a fully distributed system in every sense of the word, because spark is a strict master->slave architecture.
If we agree with that  : Then and there is no fundamental difference between 1, 2 or 3 slaves, is there?